File: app/src/main/java/com/example/myapplication/models/mangamodels/ReadMangaModel.java
package com.example.myapplication.models.mangamodels;

import java.util.ArrayList;
import java.util.List;

import lombok.Data;

@Data
public class ReadMangaModel {
    private String chapterTitle;
    private List<AllChapterDatas> allChapterDatas;
    private String nextMangaURL;
    private String previousMangaURL;
    private String mangaDetailURL;
    private List<String> imageContent = new ArrayList<>();

    @Data
    public class AllChapterDatas {
        private String chapterTitle;
        private String chapterUrl;
    }

}

File: app/src/main/java/com/example/myapplication/models/mangamodels/ReadMangaModel.kt
package com.example.myapplication.models.mangamodels


data class ReadMangaModel(
        var chapterTitle: String? = null,
        var allChapterDatas: List<AllChapterDatas>? = null,
        var nextMangaURL: String? = null,
        var previousMangaURL: String? = null,
        var mangaDetailURL: String? = null,
        var imageContent: MutableList<String>? = mutableListOf()
) {
    data class AllChapterDatas(
            var chapterTitle: String? = null,
            var chapterUrl: String? = null
    )
}
